Dockerfile

```diff
@@ -1,11 +1,16 @@
-FROM quay.io/ukhomeofficedigital/hocs-base-image
+FROM quay.io/ukhomeofficedigital/hocs-base-image:4a4baa16a08fc6cbbf4369a2c922ae9caf43cc45
```

Comment:
I'm not sure pegging to a specific version is a good idea: are you confident we'll remember to update this hash next time base-image has an update?

Reply:
Good spot, this is accidental